src/hooks/useProAmmPositionFees.ts

```diff
@@ -8,17 +8,30 @@ import { useProAmmTotalFeeOwedByPosition } from './useProAmmPreviousTicks'
 
 export function useProAmmPositionFees(tokenId?: BigNumber, position?: Position, asWETH = false) {
   const tokenIdHexString = tokenId?.toHexString()
-  const amounts = useProAmmTotalFeeOwedByPosition(position?.pool, tokenIdHexString)
-  if (position && amounts.length === 2) {
-    return [
-      CurrencyAmount.fromRawAmount(
-        !asWETH ? unwrappedToken(position?.pool.token0) : position?.pool.token0,
-        amounts[0].toString(),
-      ),
-      CurrencyAmount.fromRawAmount(
-        !asWETH ? unwrappedToken(position?.pool.token1) : position?.pool.token1,
-        amounts[1].toString(),
-      ),
-    ]
-  } else return [undefined, undefined]
+  const { current, last24h } = useProAmmTotalFeeOwedByPosition(position?.pool, tokenIdHexString)
+  if (position && current.length === 2) {
+    return {
+      current: [
+        CurrencyAmount.fromRawAmount(
+          !asWETH ? unwrappedToken(position?.pool.token0) : position?.pool.token0,
+          current[0],
+        ),
+        CurrencyAmount.fromRawAmount(
+          !asWETH ? unwrappedToken(position?.pool.token1) : position?.pool.token1,
+          current[1],
+        ),
+      ],
+
+      last24h: [
+        CurrencyAmount.fromRawAmount(
+          !asWETH ? unwrappedToken(position?.pool.token0) : position?.pool.token0,
+          last24h[0],
+        ),
+        CurrencyAmount.fromRawAmount(
+          !asWETH ? unwrappedToken(position?.pool.token1) : position?.pool.token1,
+          last24h[1],
+        ),
+      ],
+    }
+  } else return { current: [undefined, undefined], last24h: [undefined, undefined] }
```

Comment:
Wrap return values my `useMemo`

Reply:
why?
I dont need useMemo here. dont overuse it

Reply:
Checked, we dont needed it since we unwraping results